src/component/Node.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import CSS from 'csstype';
+
+function Node({x, y, label}:{x: number, y: number, label?: string}) {
+  const style: CSS.Properties = {
+    position: 'absolute',
```

Comment:
It won't be convenient in future to change the styles when we have CSS in JS but I see it might be problematic to adjust right now so let's leave it as it is for now

Reply:
It needs to be here for proper nde placemant on Canvas. Styling can still be manipulated with .node CSS class.